JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -471,12 +502,13 @@ CorrelationBayesian <- function(jaspResults, dataset=NULL, options, ...) {
 .addSupportedBfIndicators <- function(table, bf, rowNames, colNames) {
   symbol <- NULL
   
-  if (bf >= 100)
+  if (bf >= 100) {
     symbol <- "***"
-  else if (bf >= 30)
+  } else if (bf >= 30) {
     symbol <- "**"
-  else if (bf >= 10)
+  } else if (bf >= 10) {
     symbol <- "*"
+  }
```

Comment:
I see that a major part of this PR is adding `{ }` around all the single statements. I find code easier to read when braces are not included when not necessary (it's nice to be consistent with our style; what do you think @vandenman, @JorisGoosen)?

Reply:
I do this especially whenever there's an 
- else 
- else if
- nested ifs/for/while
- statements of more than one line

Previous experience has shown R code failure due to these ambiguities

Reply:
nested if: totally nice if there are braces because otherwise else becomes terribly ambiguous.

All other cases of one line: please no braces.. more typing, more mess and unnecessary.
More than one line: of course braces.

Reply:
I tend to avoid `{` whenever I can. That includes constructions like:

```r
for (x in xs)
  if (x < 0)
    # do thing
  else
    # do other thing
```
This only works if it's a single line though.

Reply:
That already introduces ambiguity to me as I don't know where exactly it ends the for loop for me + white space issues.

Reply:
Okay so it seems I'm not alone in this. I'll go ahead and merge this, because I do not considering it important enough right before a release. But as you can see this type of change is not considered to be very desired.

Reply:
And @AlexanderLyNL where is the ambiguity? How could the for loop only run the "then" part while ignoring the "else"? If it did that it would wouldn't work...